src/pages/gateway/create-mesh.md

```diff
@@ -189,6 +189,36 @@ aio api-mesh:update update-mesh.json
     }
 ```
 
+## Modify projects and workspaces
+
+<InlineAlert variant="info" slots="text"/>
+
+When creating a mesh for the first time, you must select the project and workspace that you want to create the mesh in. Alternatively, you can use the following [aio commands](https://github.com/adobe/aio-cli#commands) to select or deselect a project or workspace without interacting with your mesh.
+
+[aio commands](https://github.com/adobe/aio-cli#commands) allow you to list the current selections, manually select a project or workspace, or remove the currently selected project or workspace from the cache.
+
+### View cached project and workspace
+
+To see your current cache configuration, use the [`aio config:get:console`](https://github.com/adobe/aio-cli#aio-configget-key) command, which includes the currently selected organization, project, and workspace.
+
```

Comment:
You can view a list of available projects in your current organization by running the `aio console:project:list` command.

To view a list of available workspaces in the current project, run the `aio console:workspace:list` command.

Reply:
[This page](https://developer.adobe.com/app-builder/docs/getting_started/first_app/#2-creating-a-new-project-on-developer-console) says:

`Each App Builder project has two default workspaces: Production and Stage. You can add more workspaces as needed. The Production workspace is special, as it is used for the submission and distribution flow. When you’re ready to deploy your app, you will submit it for approval from the Production workspace.`

So maybe I should tell them to use `Stage` when in doubt.